railties/test/generators/shared_generator_tests.rb

```diff
@@ -290,15 +290,13 @@ def test_generator_if_skip_action_cable_is_given
     end
   end
 
-  def test_generator_when_sprockets_is_not_used
-    run_generator [destination_root, "-a none"]
+  def test_generator_when_asset_pipeline_is_not_used
+    run_generator [destination_root, "--skip-asset-pipeline"]
```

Comment:
`-a` is currently only an option for apps, but not plugins. If it should be an option for plugins, I can change this back and move where the `class_option` is defined